lib/logs/log_formatter.rb

```diff
@@ -42,11 +42,13 @@ def call
         if Rails.env.local?
           color, background, style = color_background_and_style(key, value)
 
-          key_value_string =
-            Rainbow(key_value_string).
-              color(color).
-              background(background || :default).
-              public_send(style || :itself)
+          if defined?(Rainbow) # Rainbow won't be defined when running w/ Docker
```

Comment:
(Unrelated change to avoid errors when running docker about Rainbow not being a method.)